client_app/src/components/volunteer/hooks/useShelterData.js

```diff
@@ -17,18 +18,18 @@ export const useShelterData = (defaultRadius) => {
 
   const fetchData = () => {
     setLoading(true);
-    const newEndpoint = `${GETHELP_API}v2/facilities?page=0&pageSize=1000&latitude=${latitude}&longitude=${longitude}&radius=${radius}`;
+    const newEndpoint = `${SERVER}/shelter`;
     
     fetch(newEndpoint, {
       method: "GET",
       headers: { "Content-Type": "application/json" }
     })
       .then(response => response.json())
       // uncomment this when we switch to using our /shelter API endpoint
-      // .then(data => {
-      //   setOriginalData(data.content);
-      //   setLoading(false);
-      // })
+       .then(data => {
+         setOriginalData(data.content);
+         setLoading(false);
+       })
```

Comment:
We are setting the shelter data retrieved from the server, but it's later getting overriden by values imported from ShiftsData, which is why we don't see the shelters defined in our system on the Volunteer dashboard.